core/src/player.rs

```diff
@@ -772,6 +775,33 @@ impl Player {
         })
     }
 
+    pub fn set_salign(&mut self, salign: &str) {
+        self.mutate_with_update_context(|context| {
+            let stage = context.stage;
+            if let Ok(salign) = StageAlign::from_str(salign) {
+                stage.set_align(context, salign);
+            }
+        })
+    }
```

Comment:
I say let's have the `Player` be the line between the Flash HTML attribute names and our internal attribute names  -- I like `set_stage_align` here.